tpl/tplimpl/template_embedded.go

```diff
@@ -84,7 +84,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
       <pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
       {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
-      <description>{{ .Content | html }}</description>
+      <description>{{ .Summary | html }}</description>
```

Comment:
@DarwinJS  Seems the description page variable with a default summary value makes more sense since it provides the user more control.

```
{{ .Description | default .Summary }}
```

Reply:
This change has broken blog aggregators that expect the full content to be available in the feed. See for example any of the feeds syndicated by:

* http://planet.debian.org/
* http://planet.fsfe.org/
* http://planet.57north.org.uk/
* http://fedoraplanet.org/

I could give more examples.

Reply:
For anyone stumbling across this from Google and looking for a workaround, I have published a fix on my blog at https://iain.learmonth.me/blog/2017/2017w396/